bigframes/ml/model_selection.py

```diff
@@ -147,3 +149,37 @@ def split(
             yield utils.convert_to_types(
                 [X_train, X_test, y_train, y_test], [X, X, y, y]
             )
+
+
+def cross_validate(
+    estimator,
+    X: Union[bpd.DataFrame, bpd.Series],
+    y: Union[bpd.DataFrame, bpd.Series, None] = None,
+    *,
+    cv: Optional[Union[int, KFold]] = None,
+) -> dict[str, list]:
+    if cv is None:
+        cv = KFold(n_splits=5)
+    elif isinstance(cv, int):
+        cv = KFold(n_splits=cv)
+
+    result: dict[str, list] = {"test_score": [], "fit_time": [], "score_time": []}
+    for X_train, X_test, y_train, y_test in cv.split(X, y):  # type: ignore
+        fit_start_time = time.time()
+        estimator.fit(X_train, y_train)
+        fit_time = time.time() - fit_start_time
+
+        score_start_time = time.time()
```

Comment:
Maybe use perf_counter. From chatgpt
time.time(): It is subject to system clock adjustments or skew. The system time can be adjusted backward and forward by the operating system, which can lead to inaccurate or unreliable results when measuring short durations or intervals. This makes it less suitable for performance testing where precise and stable measurements are critical.

Reply:
Done.